tests/samples-messaging-amqp/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-contract-tests</artifactId>
+		<version>1.0.1.BUILD-SNAPSHOT</version>
+		<relativePath>..</relativePath>
+	</parent>
+	<artifactId>spring-cloud-contract-sample-amqp</artifactId>
+	<packaging>jar</packaging>
+	<name>Spring Cloud Contract Sample Spring Amqp</name>
+	<description>Spring Cloud Contract Sample Spring Amqp Rabbit</description>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-actuator</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-amqp</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
Would we have to require from users to explicitly provide `mockito` in compile scope? That might be bizarre. I think that our starter should provide `mockito` in compile scope, whereas  the user uses our starter in test scope. Or am I missing sth?